bump_pydantic/codemods/replace_config.py

```diff
@@ -38,6 +37,7 @@
     "orm_mode": "from_attributes",
     "schema_extra": "json_schema_extra",
     "validate_all": "validate_default",
+    "allow_mutation": "frozen",
```

Comment:
Since we are going to check the same "allow_mutation" value again, there's no need to include it in this dictionary.